src/main/kotlin/com/devfloor/untitled/articlefavorite/domain/ArticleFavoriteType.kt

```diff
@@ -17,4 +17,8 @@ enum class ArticleFavoriteType(
 
     override val id: String
         get() = this.name
+
+    companion object {
+        fun from(type: String) = values().first { it.text == type }
```

Comment:
이거 매개변수 이름을 text로 가져가는 건 어때? 별론가?

```suggestion
        fun from(text: String) = values().first { it.text == text }
```

Reply:
반영할게 !!